config/settings.py

```diff
@@ -3,11 +3,11 @@
 
 from dotenv import load_dotenv
 
-load_dotenv()
-
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
 
+load_dotenv(dotenv_path=BASE_DIR / "infra/.env")

```

Comment:
```suggestion
load_dotenv(dotenv_path=BASE_DIR / "infra" / ".env")

```
В доках описано такое использование)
(см. абзац "Navigating inside a directory tree")
https://docs.python.org/3/library/pathlib.html#basic-use

Ну и выглядит чище, на мой взгляд)